app/ui/design-system/src/lib/Components/InternalSidebar/index.tsx

```diff
@@ -60,10 +60,14 @@ export const TEMP_SIDEBAR_CONFIG: InternalSidebarConfig = {
 }
 
 export function InternalSidebar({ config, menu }: InternalSidebarProps) {
+  const location = useLocation()
+  const itemOverride =
+    location.pathname === "/flow/" || location.pathname === "/nodes/"
```

Comment:
Hate to see this, but seems unavoidable with the current setup. Going tohold of on merging this until @joekrill finishes up with the routing changes.